experiment/test_measurer.py

```diff
@@ -298,6 +298,8 @@ def _create_measurer(fuzzer, benchmark, trial_num):
 class TestIntegrationMeasurement:
     """Integration tests for measurement."""
 
+    @pytest.mark.skipif(not os.getenv('FUZZBENCH_TEST_INTEGRATION'),
```

Comment:
Why disable, why not use some other benchmark, this feels like important test.

Reply:
Frankly I'm a worried there will always be some reason a random binary wont run on someone's machine, I could try a different benchmark but I'm worried well see the same issue.

Reply:
Sorry, missing some context. Are we building and running a coverage binary outside docker? If yes, would it make sense to either:
1) do it in docker, or
2) use a fake coverage binary (could be a python script that outputs what we expect)
?

Reply:
I think we essentially do 2 elsewhere, just not here because this is the integration test and should actually run the binary.
You're probably right that this should be done in docker.

Reply:
+1 docker is better, dont know how much work is this ?

Reply:
I don't think getting it to work elsewhere is a priority right now. I don't expect many people to modify the code that this test covers. I'll leave it as a TODO.